### DISCUSSION TITLE
Adding animations to GraphEdit connection lines

### DISCUSSION THREAD

**Discussion by abbas-tari** [godotengine/godot-proposals#11359] [2D]
The connection lines are solid in GraphEdit and it would be awesome if there is a chance to give more freedom in adding animations or different types/shapes of line to GraphEdit connection lines? Any idea feedback, suggestions?

**Comment by Calinou** [comment]
- Related to https://github.com/godotengine/godot-proposals/issues/11384.

Any kind of animated connection means that the GraphEdit will have to redraw every frame, which will make it much slower and greatly increase CPU/GPU requirements (and therefore power consumption). This can be OK for games if they're redrawing every frame either way, but I expect this to be a no-go for non-game applications.
